### ISSUE TITLE
Why the Dropout2d and BatchNorm2d's model.eval() result is poor

### ISSUE THREAD

**Issue by xiaopengguo1406** [pytorch/pytorch#3869]
Recent I'm doing a project using the elegant tool,pytorch. I use the dropout2d and batchnorm2d in my network.The trained model performance well in the test phase if I didn't using the model.eval(). However,when I use the model.eval(), the result is very pool. I really feel  confused.I sincerely hope the developer can help me solving this confusion. Thank you very much!

**Comment by tofigh-** [comment]
decrease momentum value in BatchNorm layer to something small like 0.0001

**Comment by soumith** [comment]
use @tofigh- 's solution.

**Comment by theonegis** [comment]
> Recent I'm doing a project using the elegant tool,pytorch. I use the dropout2d and batchnorm2d in my network.The trained model performance well in the test phase if I didn't using the model.eval(). However,when I use the model.eval(), the result is very pool. I really feel confused.I sincerely hope the developer can help me solving this confusion. Thank you very much!

Have you solved this problem by decreasing the momentum in BatchNorm layer?
